project-base/storefront/types/customer.ts

```diff
@@ -14,7 +14,7 @@ export type DeliveryAddressType = {
     telephone: string;
     firstName: string;
     lastName: string;
-    country: string;
+    country: TypeCountryFragment;
```

Comment:
What was the motivation behind this change

Reply:
The motivation behind changing the type from string to TypeCountryFragment is to better support the delivery address form, specifically the country selection. By using TypeCountryFragment, which likely includes both a label and value for each country, we can more effectively manage the country options in the select box. This change allows us to display the country name (label) to the user while storing the country code (value) internally.